### PR TITLE
CDAP-1725 Upgrade tool creates system ns in hbase

### PR DESCRIPTION
**Pull request by rohitsinha54**
- Upgrade tool creates system namespace in hbase if does not exists.

Issues: 
https://issues.cask.co/browse/CDAP-1725

Build : http://builds.cask.co/browse/CDAP-DUT1004-4

### CODE REVIEW

File: cdap-master/src/main/java/co/cask/cdap/data/tools/UpgraderMain.java [hunk 1]
```diff
@@ -340,9 +347,22 @@ private void addModules(DatasetFramework datasetFramework) throws DatasetManagem
   }
 
   /**
-   * Creates the {@link Constants#DEFAULT_NAMESPACE} namespace
+   * Creates the {@link Constants#SYSTEM_NAMESPACE} in hbase and {@link Constants#DEFAULT_NAMESPACE} namespace and also
+   * adds it to the store
    */
-  private void createDefaultNamespace() {
+  private void createNamespaces() {
```
Comment (bdmogal) [inline cdap-master/src/main/java/co/cask/cdap/data/tools/UpgraderMain.java]: Is there a chance that some of the upgrade functions (especially those related to HBase tables) are performed before this method is called?

Comment (rohitsinha54) [inline cdap-master/src/main/java/co/cask/cdap/data/tools/UpgraderMain.java]: From the upgrade code I don't see it.

Comment (bdmogal) [inline cdap-master/src/main/java/co/cask/cdap/data/tools/UpgraderMain.java]: Ok.

**Comment by bdmogal** [comment]
LGTM

**Comment by rohitsinha54** [comment]
Build running here: http://builds.cask.co/browse/CDAP-DUT1004-5
Should me merged only after build passes.

**Comment by rohitsinha54** [comment]
@abaranau:
http://builds.cask.co/browse/CDAP-DUT1004-6
testInjector passed but 
develop is broken
http://builds.cask.co/browse/CDAP-DUT-435
Merge 
?

**Comment by anwar6953** [comment]
Restarted the build, since develop is passing now.

**Comment by rohitsinha54** [comment]
@anwar6953 : Thanks. You are the man :+1:

**Comment by rohitsinha54** [comment]
Build :green_heart: http://builds.cask.co/browse/CDAP-DUT1004-7
Merging.

**Pull request merged**